.github/ISSUE_TEMPLATE/onboarding_argocd.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Onboard to Continuous Delivery
+about: Requesting application to be automatically deployed via ArgoCD to any managed OpenShift cluster
+title: ""
+labels: onboarding
+assignees: ""
+---
+
+### Requested actions
+
+Select all that apply:
+
+- [ ] Create a new ArgoCD project for existing team
+- [ ] Deploy an application
```

Comment:
ist "Create" required if I do "Deploy" ?

Reply:
Good catch.. we should change the wording here. :+1:

Reply:
What about this: `Enlist an application to be deployed automatically via ArgoCD`